xerces-c.spec

```diff
@@ -46,11 +39,11 @@ esac
 
 case %{cmsplatf} in
   slc*_amd64_*)
-    ./runConfigure -P%{i} -plinux -cgcc -x%{cms_cxx} ;;
+    ./configure --prefix=%{i};;
   *_aarch64_*|*_ppc64le_*)
-    ./runConfigure -P%{i} -b 64 -plinux -cgcc -x%{cms_cxx} ;;
-  osx*)
-    ./runConfigure -P%{i} -b 64 -pmacosx -nnative -rnone -cgcc -x%{cms_cxx} ;;
+    ./configure --prefix=%{i} --build=64;;
+ # osx*)
+ #   ./configure -P%{i} -b 64 -pmacosx -nnative -rnone -cgcc -x%{cms_cxx} ;;
```

Comment:
Remove this `case` statement and just use `./configure --prefix=%{i}`. Also note, that `--build=64` is not a correct approach. It should contain a triplet of machine you are building on. You don't need `--build` as autoconf will auto-detect it.